.flake8

```diff
@@ -3,3 +3,4 @@ exclude = ./tests/fixtures/
 
 # Match black tool's default.
 max-line-length = 88
+extend-ignore = E203
```

Comment:
this was recommended by black documentation: 
https://ichard26-testblackdocs.readthedocs.io/en/refactor_docs/the_black_code_style.html#line-length

Reply:
Nit: please add back last line end characters here and in other files.